fileutil/compressor_interface.go

```diff
@@ -4,13 +4,14 @@ type CompressorOptions struct {
 	SameOwner       bool
 	PathInArchive   string
 	StripComponents int
+	NoCompression   bool
 }
 
 type Compressor interface {
 	// CompressFilesInDir returns path to a compressed file
-	CompressFilesInDir(dir string) (path string, err error)
+	CompressFilesInDir(dir string, options CompressorOptions) (path string, err error)
```

Comment:
Does passing through the whole `CompressorOptions` mean that all consumers must also determine `SameOwner`, `PathInArchive`, and `StripComponents`?

The alternative here would be to just pass in the `NoCompression` boolean.